### ISSUE TITLE
GPU allocated

### ISSUE THREAD

**Issue by rykawamura** [mazurowski-lab/finetune-SAM#20]
Thank you for your excellent work on the paper.
I encountered a CUDA out-of-memory error with a batch size of 2 on an A100 GPU. I noticed in Table 3 that the GPU memory allocated is significantly larger than the number of trainable parameters. Could you please tell me why there is such a difference? 
Thank you.

**Comment by Guhanxue** [comment]
Hi, for this, may i ask which encoder you are using?

To take Vit-H as an example, even though the trainable parameters can be saved bc of parameter-efficient learning, the entire encoder needs to be put into GPU (bc even those parameters are not updated, but you still needs to calculate gradients going through them as they connected to other trainable Adapters during propagation). 

In this case, even the trainable parameters is siginicantly lower, but the occupied GPU size is almost the same (or even more bc you add additional Adapters) as you train all the paramters.  

Thus, to the best of my knowledge, parameter-efficient learning can mostly accelerate the training speed and prevent collapse but not help reduce the size of GPU occupation. 

There might be some other PEFTs that can save GPU size, like Ladder-network or side-network, but i think Adapter/LoRA don't belong to that category, see the reference in https://www.reddit.com/r/MachineLearning/comments/186ck5k/d_what_is_the_motivation_for_parameterefficient/.
